### DISCUSSION TITLE
I got approved by not get benifits

### DISCUSSION THREAD

**Discussion by abhilash-2005** [community/community#135740] [GitHub Education]
### Hi everyone,

I  got approved mail form GitHub and on website also but while I click on any offer its shows error 404
![Screenshot_2024-08-14-10-43-27-15_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/689ea53f-5ae4-46fb-afbd-defbec0d9dd6)
![Screenshot_2024-08-14-10-42-39-71_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/db6e0969-c684-4a30-b902-4886ce75ea58)

**Comment by GabrielAlmeidag** [comment]
Hi @abhilash-2005 ,

Yes, today the Student Developer Pack offers page is inaccessible, I guess they will fix it soon.

**Comment by Razano26** [comment]
@abhilash-2005 If you are approved, you can directly go to the page https://github.com/settings/billing/summary to check your plan.
If your Current plan is GitHub pro you can use all pro feature and get the list of GitHub Student Developer Pack there https://education.github.com/pack.

**Reply by abhilash-2005** [reply]
I understand thank you bro

**Reply by Razano26** [reply]
@abhilash-2005 With this link it work (https://education.github.com/pack), so I thinks you can solve the thread

**Comment by prabhjotschugh** [comment]
Hi
After approval, it can take about 5-7 days to receive the benefits. You will get an email regarding the same.
Hope it helps!
